core/code/portal_info.js

```diff
@@ -292,14 +292,14 @@ window.getPortalHackDetails = function(d) {
   // first mod of type is fully effective, the others are only 50% effective
   var effectivenessReduction = [ 1, 0.5, 0.5, 0.5 ];
 
-  var cooldownTime = 300; // 5 mins - 300 seconds 
+  var cooldownTime = COOLDOWN_TIME;
 
   $.each(heatsinks, function(index,mod) {
     var hackSpeed = parseInt(mod.stats.HACK_SPEED)/1000000;
     cooldownTime = Math.round(cooldownTime * (1 - hackSpeed * effectivenessReduction[index]));
   });
 
-  var numHacks = 4; // default hacks
+  var numHacks = HACK_COUNT; // default hacks
```

Comment:
Let's keep names in sync.
- Either `var numHacks = NUM_HACKS;`
- Or `var hackCount = HACK_COUNT;`

Reply:
1) AP_FACTOR, not a boolean
2) a COUNT  is something ongoing, better something like MAX_HACKS.

Reply:
MAX_HACKS sounds good